src/extensions/extra/MDNDocs/README.md

```diff
@@ -0,0 +1,7 @@
+# Updating the Docs
+Use the Node script scrape-mdn in `/tools` or clone repo from [scrape-mdn](http://github.com/jordantheriault/scrape-mdn) to update the `css.json` and 'html.json' contents:
+
+1. Navigate to `/tools`
+2. Run `npm install`
+3. Enter `node app`
+4. Copy `json/html.json` and `json/css.json` to this directory in the extension.
```

Comment:
instead just overwrite the existing files (also rename the actual files to `html.json`)